kpatch-build/kpatch-build

```diff
@@ -554,7 +554,10 @@ if [[ $CHANGED -eq 0 ]]; then
 fi
 
 echo -n "Patched objects:"
-for i in "${!objnames[@]}"; do echo -n " $(basename $i)"; done
+for i in $(echo "${objnames[@]}" | tr ' ' '\n' | sort -u | tr '\n' ' ')
```

Comment:
@libin2015  ah you are right.  i was thinking about the uniq command.  sort -u keeps the last occurrence if there are duplicates (for the record).